views/scss/qtiCreator/_choice.scss

```diff
@@ -57,6 +57,7 @@
                 @include rtl {
                 margin-left: 2.25em;
                 span.mini-tlb {
+                        margin-top: -0.15em;
```

Comment:
Do you think this is necessary?
![image](https://user-images.githubusercontent.com/11692751/159685603-9da121ee-4fdc-4f0d-8113-8b5f40a0101d.png)

Reply:
I added because it doesn't look well when adding data to math expression... but I can see that in empty item doesn't look well... 🤔 not sure what's the best way...